libraries/server_scripts.rb

```diff
@@ -71,11 +71,12 @@ def configure
         --console
       <%= catch_powershell_error('Configuring Home Dir') %>
 
-      .\\Octopus.Server.exe configure `
+      .\\Octopus.Server.exe database `
         <%= option('instance', resource.instance) %> `
-        <%= option('storageConnectionString', resource.connection_string) %> `
+        <%= option('connectionString', resource.connection_string) %> `
+        <% if resource.create_database %>--create<% end %> `
```

Comment:
Can you use the following for this option. 
https://github.com/cvent/octopus-deploy-cookbook/blob/master/libraries/shared.rb#L37-L40